README.md

```diff
@@ -108,6 +108,12 @@ resilientRawRepresentableProperty
 
 **Note:** One difference the errors available on the property wrapper and those reported to the `ResilientDecodingErrorReporter`, is the latter _does not_ report `UnknownNovelValueError`s by default (`UnknownNovelValueError` is thrown when a non-frozen `ResilientRawRepresentable`'s `init(rawValue:)` returns `nil`). You can alter this behavior by calling `errors(includeUnknownNovelValueErrors: true)` on the error digest. 
 
+## Frequently Asked Questions
+
+### Will `Resilient` work with generic type arguments?
```

Comment:
Could we be more specific with what “work” means? Maybe “How does `Resilient` decode collections when the collection is a hidden by a generic type argument?